Collapsible menu

VoodLyc opened the pull request:
Add collapsible menu functionality

VoodLyc committed:
The menu is collapsible

VoodLyc committed:
Fix menu size

VoodLyc committed:
Navbar button style changes onClick

VoodLyc committed:
Refactor CSS class names

rodrodogo commented on src/index.html:
```diff
@@ -7,55 +7,56 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <link href='https://css.gg/pokemon.css' rel='stylesheet'>
```
Remove unused link

rodrodogo commented on src/app.js:
```diff
@@ -0,0 +1,41 @@
+}
+
+const selectButton = (e) => {
+    if (e.target.classList.contains('navbar-button')) {
```
Improve readability

rodrodogo commented on src/app.js:
```diff
@@ -0,0 +1,41 @@
+const selectButton = (e) => {
+    if (e.target.classList.contains('navbar-button')) {
+        if (selectedButton != e.target.id) {
+            e.target.classList.toggle('navbar-button--selected')
```
This section could be reused with a method

rodrodogo commented on src/app.js:
```diff
@@ -0,0 +1,41 @@
+let appName = document.querySelector('.menu__name__label')
+let buttonsContainer = document.querySelector('.navbar__buttons-container')
+let selectedButton = 'navbar-btn-1'
+
```
Remove empty space

VoodLyc committed:
Refactor code

- Changed the order of CSS classes to match the HTML.
- Changed document name.
- Added a favicon.
- Changed the arrow button (the button that collapses the side menu) from div to button.

VoodLyc committed:
Fix scroll

The screen now uses the entire space without the need of scrolling. Also added
some ARIA concepts in the HTML.
